content/blog/2020-03-12-native-classroom/index.Rmarkdown

```diff
@@ -0,0 +1,93 @@
+---
+title: "Data Analysis for Social Justice in a Native Classroom"
+date: '2020-03-12'
+slug: native-classroom
+categories:
+  - teach
+author:
+  - "[Nic Thompson Gonzalez](http://www.nicolethompsongonzalez.com/)"
+photo:
+  URL: https://unsplash.com/photos/3KEJMfoHoMY
+  author: Graeme Nicholl
+---
+
+In this post, I'll share some insights from a work in progress. Last year, I began teaching an introduction to data science at the Southwestern Indian Polytechnic Institute [(SIPI)](https://www.sipi.edu/), a community college in Albuquerque, New Mexico with an exclusively Native American student-body. Although my professional background is research in behavioral ecology, my teaching passion is for data analysis. Why? Because I think data literacy and analytical expertise is one freely available tool to remedy many societal ills and injustices. My course datasets are therefore usually related to these ills. The material is currently delivered in an intensive 4-day workshop. One day, I hope it can be offered as a full course.
+
+From the first round of developing and teaching the workshop, I’ve learned a number of lessons (beyond how to teach mental models that underlie ggplots). I can group these into 3 general areas.
+
+1. Creating culturally relevant and inclusive content and structure
+  
+2. The importance of logistics and technology
+  
+3. A teaching attitude that’s make-or-break for beginners
+
+
+# **1.	Creating culturally relevant and inclusive content and structure**
+
+In an inclusive classroom, we’re addressing two big issues. First – is the material personally relevant? Second – is the material accessible and safe to engage with? ^[[Guide for Inclusive Teaching at Columbia](https://cpb-us-w2.wpmucdn.com/edblogs.columbia.edu/dist/8/1109/files/2020/02/Guide-for-Inclusive-Teaching-at-Columbia_Accessibility-Revisions_15-January-2020_FINAL.pdf)] 
```

Comment:
I would write this as, "An inclusive class must address two big issue: is the material personally relevant, and is the material accessible and safe to engage with?"